generator/generator.yml

```
@@ -710,3 +710,24 @@ modules:
         type: EnumAsInfo
       "1.3.6.1.4.1.534.6.6.7.6.1.1.3":
         type: DisplayString
+
+# CyberPower
+#
+# https://www.cyberpowersystems.com/product/software/mib-files/mib-v29/
+  cps_ups:
```

This should probably be `cyberpower` to make it more clear.

```suggestion
  cyberpower:
```